backend/src/db/migrations/20230824185213_add_meditation_entries_table.ts

```diff
@@ -1,6 +1,6 @@
 import { type Knex } from 'knex';
```

Comment:
remove changes in this file
1. we shouldn't modify old migrations
2. we shouldn't add external dependencies like DatabaseTableName to migrations